masschallenge/accelerator-directory
# [AC-5059] - Build a public directory that searches Organization

**Changes Introduced in [AC-5059](https://masschallenge.atlassian.net/browse/AC-5059)**

- A new MVP repo, that uses a local (configurable) impact-api server to search for organization names, and link to their local accelerate Organization page.

**Test:**
1. on Accelerate, checkout AC-5059, and `runserver`
1. on impact-api, checkout AC-5059 and `make build;make dev`
1. Follow the instructions on the README.md on this PR, in order to install, configure, and run accelerator-directory.
1. Use the interface to find the organization that you want.
1. Click on one of the link results and see the organization page in accelerate (after you log in)
